tests/director/tests/application_api_test.go

```diff
@@ -493,7 +493,23 @@ func TestUpdateApplicationWithLocalTenantIDShouldBeAllowedOnlyForIntegrationSyst
 	updatedApp := graphql.ApplicationExt{}
 
 	t.Run("should fail for non-integration system", func(t *testing.T) {
-		err = testctx.Tc.RunOperation(ctx, certSecuredGraphQLClient, request, &updatedApp)
+		runtime, err := fixtures.RegisterRuntime(t, ctx, certSecuredGraphQLClient, "test-runtime", tenant.TestTenants.GetDefaultTenantID())
```

Comment:
Can you elaborate on why this change is necessary? From the test name, I assume the old code with the test run should fail as well? because it uses certificate creds instead of an integration system?

Reply:
Thanks for the clarification!